components/utilities/ymodem/ymodem.h

```diff
@@ -138,7 +145,30 @@ struct rym_ctx
  * second.
  */
 rt_err_t rym_recv_on_device(struct rym_ctx *ctx, rt_device_t dev, rt_uint16_t oflag,
-        rym_callback on_begin, rym_callback on_data, rym_callback on_end,
-        int handshake_timeout);
+                            rym_callback on_begin, rym_callback on_data, rym_callback on_end,
+                            int handshake_timeout);
+
+/** send a file on device dev with ymodem session ctx.
```

Comment:
Please notice the comments of `/**`, which is a doxygen comments to generate documentation. However, the other of this file, there is no doxygen comments. Please keep style.